dev-libs/softhsm/softhsm-2.3.0.recipe

```diff
@@ -0,0 +1,85 @@
+SUMMARY="An implementation of a cryptographic store"
+DESCRIPTION="SoftHSM is an implementation of a cryptographic store \
+accessible through a PKCS #11 interface. You can use it to explore PKCS #11 \
+without having a Hardware Security Module. It is being developed as a part \
+of the OpenDNSSEC project. SoftHSM uses Botan for its cryptographic \
+operations."
+HOMEPAGE="https://www.opendnssec.org/softhsm/"
+COPYRIGHT="2010 .SE, The Internet Infrastructure Foundation and SURFnet bv"
+LICENSE="BSD (2-clause)"
+REVISION="1"
+SOURCE_URI="https://dist.opendnssec.org/source/softhsm-$portVersion.tar.gz"
+CHECKSUM_SHA256="5ed604c89a3a6ef9d7d1ee92c28a2c4b3cd1f86f302c808e2d12c8f39aa2c127"
+SOURCE_DIR="softhsm-$portVersion"
+PATCHES="softhsm-$portVersion.patchset"
+
+ARCHITECTURES="!x86_gcc2 ?x86 x86_64"
+SECONDARY_ARCHITECTURES="x86"
+
+commandBinDir=$binDir
+commandSuffix=$secondaryArchSuffix
+if [ "$targetArchitecture" = x86_gcc2 ]
+then
+	commandBinDir=$prefix/bin
+	commandSuffix=
+fi
+
+PROVIDES="
+	softhsm_2$secondaryArchSuffix = $portVersion
+	cmd:softhsm2_dump_file$commandSuffix = $portVersion
+	cmd:softhsm2_keyconv$commandSuffix = $portVersion
+	cmd:softhsm2_util$commandSuffix = $portVersion
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	botan$secondaryArchSuffix
+	lib:libssl$secondaryArchSuffix
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	"
+BUILD_PREREQUIRES="
+	botan${secondaryArchSuffix}_devel
```

Comment:
We prefer to use the `devel:libbotan_2` syntax rather than `devel:libbotan`. BTW, it needs to go to `BUILD_REQUIRES`.
```diff
-	botan${secondaryArchSuffix}_devel
+	devel:libbotan_2$secondaryArchSuffix
```